test/core/web/conn_test.exs

```diff
@@ -0,0 +1,37 @@
+# Copyright(c) 2015-2023 ACCESS CO., LTD. All rights reserved.
+
+defmodule AntikytheraCore.ConnTest do
+  use Croma.TestCase
+
+  alias Antikythera.Conn, as: LibConn
+
+  def make_fake_conn(status, body) do
+    %LibConn{
+      request: nil,
+      context: nil,
+      status: status,
+      resp_headers: %{},
+      resp_cookies: %{},
+      resp_body: body,
+      before_send: [],
+      assigns: %{}
+    }
+  end
+
+  describe "validate/1" do
```

Comment:
There is no test to check that a response with a status which must not have a body is valid when the body is actually empty.

Reply:
You're right.
Add test cases for empty body
https://github.com/access-company/antikythera/pull/234/commits/94f69c3576f8310460adfa70a8aaddb34be610d3